app/section/sections/browser-based/browser-additional-information/ui.test.js

```diff
@@ -1,186 +1,17 @@
-import nock from 'nock';
-import { Selector, ClientFunction } from 'testcafe';
-import dashboardWithCompleteSections from '../../../../../fixtures/dashboardWithCompleteSections.json';
 
-const additionalInformationData = {
-  'additional-information': 'The solution additional information',
-};
+import { runTestSuite } from '../../../../test-utils/runTestSuite';
 
-const mocks = (withMarketingData) => {
-  if (withMarketingData) {
-    nock('http://localhost:8080')
-      .get('/api/v1/Solutions/S100000-001/sections/browser-additional-information')
-      .reply(200, additionalInformationData);
-  } else {
-    nock('http://localhost:8080')
-      .get('/api/v1/Solutions/S100000-001/sections/browser-additional-information')
-      .reply(200, {});
-  }
-};
+const sectionId = 'browser-additional-information';
+const clientUrl = `http://localhost:1234/solution/S100000-001/dashboard/browser-based/section/${sectionId}`;

```

Comment:
can string interpolate the dashboardId here

Reply:
I have just removed it and now gets generated in the tests